### DISCUSSION TITLE
Regarding Scholarships

### DISCUSSION THREAD

**Discussion by sab1a** [snigi-gupta/AttendingUniversityAtBuffalo#60] [Q&A] [answered]
Does UB have any scholarships for:

- Performing well in the semesters? 
- Diversity in STEM? (Specifically for women studying STEM fields)

If so, Where can I find more information on them?

**Comment by snigi-gupta** [comment, marked as answer]
@sab1a 

I don't think so we do have these for students on F1 visa. I am not sure if you are on one. I would rather contact the grad/undergrad coordinator and ask the specific details from them. Also keep an eye to the emails sent out to you via your department and read all emails throughly, coz usually that is where such opportunities are communicated.

I apologize for the late response. (I was on a tour)
Please follow me on https://www.instagram.com/yourtechdiary/ to show your support!
